src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+[    2.511953] pci 0000:00:18.5: Adding to iommu group 12 
+[    2.511969] pci 0000:00:18.6: Adding to iommu group 12 
+[    2.511989] pci 0000:00:18.7: Adding to iommu group 12 
+[    2.512029] pci 0000:01:00.0: Adding to iommu group 13 
+[    2.512057] pci 0000:01:00.1: Adding to iommu group 13 
+[    2.512083] pci 0000:01:00.2: Adding to iommu group 13 
+[    2.512098] pci 0000:02:00.0: Adding to iommu group 13 
+[    2.512115] pci 0000:02:02.0: Adding to iommu group 13 
+[    2.512129] pci 0000:02:03.0: Adding to iommu group 13 
+[    2.512143] pci 0000:02:04.0: Adding to iommu group 13 
+[    2.512162] pci 0000:03:00.0: Adding to iommu group 13 
+[    2.512181] pci 0000:04:00.0: Adding to iommu group 13 
+[    2.512198] pci 0000:05:00.0: Adding to iommu group 13 
+[    2.512260] pci 0000:07:00.0: Adding to iommu group 14 
+[    2.512296] pci 0000:07:00.1: Adding to iommu group 14 
+[    2.512335] pci 0000:08:00.0: Adding to iommu group 15 
+[    2.512359] pci 0000:08:00.1: Adding to iommu group 15 
+[    2.512386] pci 0000:09:00.0: Adding to iommu group 16 
+[    2.512410] pci 0000:09:00.2: Adding to iommu group 17 
+[    2.512433] pci 0000:09:00.3: Adding to iommu group 18 
+[    2.512455] pci 0000:0a:00.0: Adding to iommu group 19 
+[    2.512479] pci 0000:0a:00.2: Adding to iommu group 20 
+[    2.512503] pci 0000:0a:00.3: Adding to iommu group 21 
+[    2.512712] pci 0000:00:00.2: AMD-Vi: Found IOMMU cap 0x40 
+[    2.513088] perf/amd_iommu: Detected AMD IOMMU #0 (2 banks, 4 counters/bank). 
+[    2.730288] AMD-Vi: AMD IOMMUv2 driver by Joerg Roedel <jroedel@suse.de> 
+```
+
+On an Intel system, you may get an output like the following (courtesy of the
+archwiki):
+
+```
+[    0.000000] ACPI: DMAR 0x00000000BDCB1CB0 0000B8 (v01 INTEL  BDW      00000001 INTL 00000001)
+[    0.000000] Intel-IOMMU: enabled
+[    0.028879] DMAR: IOMMU 0: reg_base_addr fed90000 ver 1:0 cap c0000020660462 ecap f0101a
+[    0.028883] dmar: IOMMU 1: reg_base_addr fed91000 ver 1:0 cap d2008c20660462 ecap f010da
+[    0.028950] IOAPIC id 8 under DRHD base  0xfed91000 IOMMU 1
+[    0.536212] DMAR: No ATSR found
+[    0.536229] IOMMU 0 0xfed90000: using Queued invalidation
+[    0.536230] IOMMU 1 0xfed91000: using Queued invalidation
+[    0.536231] IOMMU: Setting RMRR:
+[    0.536241] IOMMU: Setting identity map for device 0000:00:02.0 [0xbf000000 - 0xcf1fffff]
+[    0.537490] IOMMU: Setting identity map for device 0000:00:14.0 [0xbdea8000 - 0xbdeb6fff]
+[    0.537512] IOMMU: Setting identity map for device 0000:00:1a.0 [0xbdea8000 - 0xbdeb6fff]
+[    0.537530] IOMMU: Setting identity map for device 0000:00:1d.0 [0xbdea8000 - 0xbdeb6fff]
+[    0.537543] IOMMU: Prepare 0-16MiB unity mapping for LPC
+[    0.537549] IOMMU: Setting identity map for device 0000:00:1f.0 [0x0 - 0xffffff]
+[    2.182790] [drm] DMAR active, disabling use of stolen memory
+```
+
+## Isolating the GPU
+
+Next, we need to isolate the GPU to use the `vfio-pci` rather than the original
```

Comment:
Suggest just something like: "Make the GPU use ..." (cf. [the Handbook style guide](https://docs.voidlinux.org/contributing/void-docs/style-guide.html))